api_gateway/outputs.tf

```diff
@@ -0,0 +1,17 @@
+output "api_gateway" {
+  value = module.api_gateway
```

Comment:
Can you add a description with the link to <https://registry.terraform.io/modules/terraform-aws-modules/apigateway-v2/aws/latest?tab=outputs>?